api/db-cdr/changelog/db.changelog-2.xml

```diff
@@ -20,7 +22,7 @@
       <column name="code" type="VARCHAR(80)">
         <constraints nullable="true"/>
       </column>
-      <column name="name" type="VARCHAR(255)">
+      <column name="name" type="VARCHAR(4000)">
```

Comment:
Note that this won't get run for existing CDR DBs... if it's important for it to run, either put it in a separate changelog entry or tell everyone to reset their DB when this gets merged.

Reply:
This always runs... runAlways="true" in changset.

Reply:
Cpt criteria tree load:
changelog-6::brianfreeman ran successfully in 984ms

Reply:
OK. I think at some point prior to launch we're likely to do a rationalization of our changelog (like creating one changelog entry per schema), stop using runAlways="true", and have new changelog entries whenever we want to modify the schema. (It's hard to test migrations properly in non-prod environments otherwise.)

For now, this works great.